cmd/server/main.go

```
@@ -34,9 +35,18 @@ func main() {
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "could not load data into database: %s\n", err)
 		os.Exit(1)
 	}
+
+	averages, err := db.GetMovingAverages(ctx, pool)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "could not get moving averages: %s\n", err)
+		os.Exit(1)
+	}
+
+	analyzer.Analyze(averages)
```

golangci-lint (1.62.2)

Error return value of `analyzer.Analyze` is not checked

(errcheck)